docs/component-guides/kserve.md

```diff
@@ -1,30 +1,37 @@
+5. Verified that your current directory is the root of the repository by running the `pwd` command. The output should be `<path/to/kubeflow-manifests>` directory.
 
-### Configure Knative Serving's domain
 
-KFServing uses Knative Serving for various things like serverless deployment, autoscaling, setting up network routing resources etc. For purpose of this guide, we will focus on the routes. The fully qualified domain name(FQDN) for a route in Knative Serving by default is `{route}.{namespace}.{default-domain}`. Knative Serving routes use `example.com` as the default domain ([Reference](https://knative.dev/docs/serving/using-a-custom-domain/#default-domain-name-settings)). For example, if you create an `InferenceService` resource called `sklearn-iris` in `staging` namespace. Its URL would be `http://sklearn-iris.staging.example.com`. This is also the default configuration in Knative Serving component deployed as part of your Kubeflow deployment. You will need to configure the default domain as per your deployment. Let's see how:
+## Configure a default domain with KNative Serving
 
-As you understand from the [background](#background) section, we need to use a custom domain to configure HTTPS with load balancer. We recommend using HTTPS to to enable traffic encryption between the clients and your load balancer. So, if you are using `platform.example.com` domain to host Kubeflow, you will need to edit the `config-domain` ConfigMap in the `knative-serving` namespace to configure the `platform.example.com` to be used as the domain for the routes.
+Use [Knative Serving](https://knative.dev/docs/serving/) to set up network routing resources.
 
-**Follow the instructions** in the [procedure section of the using a custom domain guide](https://knative.dev/docs/serving/using-a-custom-domain/#procedure) on the Knative Serving documentation to configure the default domain used by Knative. You would remove the `_example` key and replace `example.com` with your domain e.g. `platform.example.com`. Following is a sample:
+The default fully qualified domain name (FQDN) for a route in Knative Serving is `{route}.{namespace}.{default-domain}`. Knative Serving routes use `example.com` as the default domain. For example, if you create an `InferenceService` resource called `sklearn-iris` in the `staging` namespace, the default domain would be `http://sklearn-iris.staging.example.com`.
 
+We recommend using HTTPS to enable traffic encryption between the clients and your Load Balancer. For example, if you use the `platform.example.com` domain to host Kubeflow, you will need to edit the `config-domain` ConfigMap in the `knative-serving` namespace to configure the `platform.example.com` to be used as the domain for the routes.
+
+Edit the ConfigMap to change the default domain as per your deployment. Remove the `_example` key and replace `example.com` with your domain (e.g. `platform.example.com`). 
```

Comment:
> We recommend using HTTPS to enable traffic encryption between the clients and your Load Balancer. For example, if you use the `platform.example.com` domain to host Kubeflow, you will need to edit the `config-domain` ConfigMap in the `knative-serving` namespace to configure the `platform.example.com` to be used as the domain for the routes.

The first sentence is true but its worded in a way that implies the next sentence shows how to use HTTPS. I think we can remove that sentence since its more of a concern surrounding ALBs in general than Kserve

Suggested rewording:

The default fully qualified domain name (FQDN) for a route in Knative Serving is `{route}.{namespace}.{default-domain}`. Knative Serving routes use `example.com` as the default domain. If you create an `InferenceService` resource called `sklearn-iris` in the `staging` namespace without changing the default domain, the resulting `InfererenceService` domain would be `http://sklearn-iris.staging.example.com`.

To host an `InferenceService` on the same domain you use to host Kubeflow, for example `platform.example.com`, you will need to edit the `config-domain` ConfigMap in the `knative-serving` namespace to configure the `platform.example.com` to be used as the domain for the routes.

Edit the ConfigMap to change the default domain as per your deployment. Remove the `_example` key and replace `example.com` with your domain (e.g. `platform.example.com`).